wandb/sdk/internal/file_stream.py

```diff
@@ -45,7 +45,7 @@ def process_chunks(self, chunks):
                     util.to_human_size(util.MAX_LINE_SIZE),
                     util.to_human_size(len(chunk.data)),
                 )
-                wandb.termerror(msg, repeat=False)
+                wandb.termerror(msg, repeat=False)  # todo: don't want logging from background
```

Comment:
for completeness re: not logging from the background process we should be moving all of these to a similar framework